src/dotnet/commands/dotnet-new/CSharp_Console1.1/Program.cs

```diff
@@ -1,9 +1,12 @@
 using System;
 
-class Program
+namespace ConsoleApp
```

Comment:
We explicitly removed the namespace in the console app template.  Was the decision to add it back?  /cc @blackdwarf